### DISCUSSION TITLE
Gathering end user feedback for TAGs and TAB

### DISCUSSION THREAD

**Discussion by joshgav** [cncf/toc#1204] [Ideas]
CNCF recently formed a Technical Advisory Board (TAB) for end users to share learnings and feedback with each other and with project maintainers. The existing Technical Advisory Groups (TAGs) for project maintainers also seek to gather end user feedback and share with TAG members and project maintainers to better determine what projects should be part of the Foundation and what solutions and patterns need to be developed.

How can we coordinate TAB and TAGs such that end users' feedback reaches TAGs and projects quickly? Would it make sense to recommend to TAB members to join and participate in relevant TAGs as they discover particular needs? Or perhaps the TAB could produce a summary report from each of its sessions and share it with TAGs?

Let's coordinate TAB and TAGs so end users see results from their feedback quickly!

**Comment by TheFoxAtWork** [comment]
@rochaporto  would you provide a quick update here so we can close the discussion?

**Reply by rochaporto** [reply]
Sure. I wrote in https://github.com/cncf/toc/discussions/1220#discussioncomment-7843039 what might be a good start to launch this conversation, i think it's also valid in this case.

@joshgav should we coordinate to start discussing this in January when the TAB kicks off?

**Reply by joshgav** [reply]
+1, seems quite similar to #1220. I'm happy to help @rochaporto form methods of collaboration from TAB to TOC/TAGs.

IMO whenever it says "TOC" in the TAB charter it could be replaced with "TOC/TAGs" as TAGs are an extension of TOC.
